src/pages/workspace/categories/ImportedCategoriesPage.tsx

```diff
@@ -0,0 +1,147 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import React, {useCallback, useState} from 'react';
+import {useOnyx} from 'react-native-onyx';
+import ConfirmModal from '@components/ConfirmModal';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import type {ColumnRole} from '@components/ImportColumn';
+import ImportSpreadsheetColumns from '@components/ImportSpreadsheetColumns';
+import ScreenWrapper from '@components/ScreenWrapper';
+import useLocalize from '@hooks/useLocalize';
+import usePolicy from '@hooks/usePolicy';
+import {closeImportPage} from '@libs/actions/ImportSpreadsheet';
+import {importPolicyCategories} from '@libs/actions/Policy/Category';
+import {findDuplicate, generateColumnNames} from '@libs/importSpreadsheetUtils';
+import Navigation from '@libs/Navigation/Navigation';
+import type {SettingsNavigatorParamList} from '@libs/Navigation/types';
+import {isControlPolicy} from '@libs/PolicyUtils';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import type SCREENS from '@src/SCREENS';
+import type {Errors} from '@src/types/onyx/OnyxCommon';
+
+type ImportedCategoriesPageProps = StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.CATEGORIES_IMPORTED>;
+function ImportedCategoriesPage({route}: ImportedCategoriesPageProps) {
+    const {translate} = useLocalize();
+    const [spreadsheet] = useOnyx(ONYXKEYS.IMPORTED_SPREADSHEET);
+    const [isImportingCategories, setIsImportingCategories] = useState(false);
+    const {containsHeader} = spreadsheet ?? {};
+    const [isValidationEnabled, setIsValidationEnabled] = useState(false);
+    const policyID = route.params.policyID;
+    const policy = usePolicy(policyID);
+    const columnNames = generateColumnNames(spreadsheet?.data?.length ?? 0);
+
+    const isControl = isControlPolicy(policy);
+
+    const getColumnRoles = (): ColumnRole[] => {
+        const roles = [];
+        roles.push(
+            {text: translate('common.ignore'), value: CONST.CSV_IMPORT_COLUMNS.IGNORE},
+            {text: translate('common.name'), value: CONST.CSV_IMPORT_COLUMNS.NAME, isRequired: true},
+            {text: translate('common.enabled'), value: CONST.CSV_IMPORT_COLUMNS.ENABLED, isRequired: true},
```

Comment:
The `enabled` column is not required.https://github.com/Expensify/App/issues/67233